mailchimp3/entities/listmemberevents.py

```diff
@@ -0,0 +1,74 @@
+# coding=utf-8
+"""
+The List Member Events API endpoint
+
+Documentation: https://mailchimp.com/developer/reference/lists/list-members/list-member-events/
+"""
+from __future__ import unicode_literals
+
+from mailchimp3.baseapi import BaseApi
+from mailchimp3.helpers import check_subscriber_hash
+
+
+class ListMemberEvents(BaseApi):
+    """
+    Use the Events endpoint to collect website or in-app actions and trigger targeted automations.
+    """
+    def __init__(self, *args, **kwargs):
+        """
+        Initialize the endpoint
+        """
+        super(ListMemberEvents, self).__init__(*args, **kwargs)
+        self.endpoint = 'lists'
+        self.list_id = None
+        self.subscriber_hash = None
+
+    def create(self, list_id, subscriber_hash, data):
+        """
+        Add an event for a list member.
+
+        :param list_id: The unique id for the list.
+        :type list_id: :py:class:`str`
+        :param subscriber_hash: The MD5 hash of the lowercase version of the
+          list member’s email address.
+        :type subscriber_hash: :py:class:`str`
+        :param data: The request body parameters
+        :type data: :py:class:`dict`
+        data = {
+            "name": string*, (Must be 2-30 characters in length)
+            "properties": object (An optional list of properties)
```

Comment:
Documentation: Limit the documentation of data attributes in the docstring to only those that are required by the endpoint. Remove "properties", "is_syncing", and "occured_at" from this docstring.